src/sentry/conf/server.py

```diff
@@ -1548,3 +1554,29 @@ def get_sentry_sdk_config():
 SENTRY_USER_PERMISSIONS = (
     'broadcasts.admin',
 )
+
+KAFKA_CLUSTERS = {
+    'default': {
+        'bootstrap.servers': 'localhost:9092',
+        'message.max.bytes': 50000000,  # 50MB, default is 1MB
+    }
+}
+
+KAFKA_PREPROCESS = 'events-preprocess'
+KAFKA_PROCESS = 'events-process'
+KAFKA_SAVE = 'events-save'
```

Comment:
Do we want to include the `events` topic (currently specified by a default in the `KafkaEventStream` constructor) here as well so that this config is a single source of truth for all the kafka topics we are using?

Reply:
Yeah for sure, I'll see if it's a small enough change to fit in this PR.